Gems/PhysX/Code/Tests/PhysXMeshColliderComponentModeTests.cpp

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <TestMeshColliderComponent.h>
+
+#include <AzManipulatorTestFramework/AzManipulatorTestFrameworkTestHelpers.h>
+#include <AzToolsFramework/UnitTest/AzToolsFrameworkTestHelpers.h>
+#include <AzToolsFramework/ViewportSelection/EditorSelectionUtil.h>
+#include <AzToolsFramework/Entity/EditorEntityHelpers.h>
+
+namespace UnitTest
+{
+    class PhysXMeshColliderComponentModeTest
+        : public ToolsApplicationFixture<false>
+    {
+    protected:
+        using EntityPtr = AZ::Entity*;
```

Comment:
Is there any value in this alias? It's only 2 characters shorter. It would make sense to me if it was e.g. a unique_ptr or something, but this seems kind of redundant.

Reply:
Copy/Paste from `PhysXColliderComponentModeTest`. But I totally agree, I'll remove it in both.